### DISCUSSION TITLE
System.Drawing namespace duplication

### DISCUSSION THREAD

**Discussion by CharliePoole** [statiqdev/Discussions#30] [Q&A] [answered]
I have the following message, arising from use of the `System.Drawing.Image` class in a `.cshtml` file:

```
The type 'Image' exists in both 'CoreCompat.System.Drawing, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' and 'System.Drawing.Common, Version=4.0.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
```

The error occurs when I do `dotnet run` after a successful build.

My bootstrap project is only referencing System.Web. I have been targeting .NET 5.0 but changing it to .NET Core 3.1 didn't get rid of the error. Does anyone know what's pulling in the conflicting references?

**Comment by daveaglick** [comment, marked as answer]
Oh no, binding problems!

A quick look reveals that this might be caused by conflicting references from Statiq.Less of all libraries:

![image](https://user-images.githubusercontent.com/1020407/98616837-9e089480-22cb-11eb-9e63-9d1971732a2b.png)

I don't even know why dotless would be pulling in drawing libraries, unless it's one of those things where using .NET Standard brings in a whole bunch of placeholder assemblies. This looks like a bug in any case, so I'm going to move it to Statiq.Framework and we can continue to work it there.

**Reply by daveaglick** [reply]
See https://github.com/statiqdev/Statiq.Framework/issues/147
